content/material/2019-11-15-intro-dplyr.Rmd

```diff
@@ -49,14 +49,16 @@ install.packages("dplyr")
 We'll be using a very simple dataset that is built-in to the tidyr package. This dataset contains data on tuberculosis cases in Afghanistan, Brazil, and China from 1999 and 2000.
 
 ```{r}
-library(dplyr)
+library(tidyverse)
 
 names(table1)
 
 str(table1)
 
 summary(table1)
 
+View(table1)
```

Comment:
Put this in comments, View shouldn't be used inside an R Markdown document, only interactively.